onnxruntime/core/framework/execution_providers.h

```diff
@@ -24,31 +24,15 @@ namespace onnxruntime {
 Class for managing lookup of the execution providers in a session.
 */
 class ExecutionProviders {
- public:
-  ExecutionProviders() = default;
-
-  common::Status Add(const std::string& provider_id, const std::shared_ptr<IExecutionProvider>& p_exec_provider) {
-    // make sure there are no issues before we change any internal data structures
-    if (provider_idx_map_.find(provider_id) != provider_idx_map_.end()) {
-      auto status = ORT_MAKE_STATUS(ONNXRUNTIME, FAIL, "Provider ", provider_id, " has already been registered.");
-      LOGS_DEFAULT(ERROR) << status.ErrorMessage();
-      return status;
-    }
-
-    // index that provider will have after insertion
-    auto new_provider_idx = exec_providers_.size();
-
-    ORT_IGNORE_RETURN_VALUE(provider_idx_map_.insert({provider_id, new_provider_idx}));
-
-    // update execution provider options
-    auto providerOptions = p_exec_provider->GetProviderOptions();
-    exec_provider_options_[provider_id] = providerOptions;
-
 #ifdef _WIN32
-    LogProviderOptions(provider_id, providerOptions, false);
+  WindowsTelemetry::EtwInternalCallback etw_callback_;
```

Comment:
nit: can we move this to the end of the class with all the other member declarations?